### PR TITLE
Feature/56 implement users list at moderator page

### PR DESCRIPTION
**Pull request by martyniukroman**

### CODE REVIEW

File: InternetHospital.BusinessLogic/Interfaces/IUserListService.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using InternetHospital.BusinessLogic.Models;
+using System.Collections.Generic;
+
+namespace InternetHospital.BusinessLogic.Interfaces
+{
+    public interface IUserListService
+    {
+        List<UserModel> GetUsers();
```
Comment (TREN-D) [inline InternetHospital.BusinessLogic/Interfaces/IUserListService.cs]: Rename it to IEnumerable

File: InternetHospital.BusinessLogic/Interfaces/IUserListService.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        (List<UserModel> users, int count) FilteredUsers(UserSearchParameters queryParameters);
```
Comment (TREN-D) [inline InternetHospital.BusinessLogic/Interfaces/IUserListService.cs]: Here too

File: InternetHospital.BusinessLogic/Services/UserListService.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            if (queryParameters.SearchByStatus != null)
+            {
+                users = users.Where(d => d.StatusId == queryParameters.SearchByStatus);
+            }
+
+            var usersQuantity = users.Count();
+            var usersResult = PaginationHelper<User>
+                .GetPageValues(users, queryParameters.PageCount, queryParameters.Page)
+                .OrderBy(x => x.SecondName)
+                .Select( x => new UserModel {
```
Comment (TREN-D) [inline InternetHospital.BusinessLogic/Services/UserListService.cs]: Under this statement you are using mapper, why here you have emitted him?

Comment (martyniukroman) [reply InternetHospital.BusinessLogic/Services/UserListService.cs]: Auto mapper isn't usable here

File: InternetHospital.BusinessLogic/Interfaces/IUserListService.cs (hunk 2 above)
Comment (martyniukroman) [reply InternetHospital.BusinessLogic/Interfaces/IUserListService.cs]: done

File: InternetHospital.BusinessLogic/Services/UserListService.cs (hunk 3 above)
Comment (martyniukroman) [reply InternetHospital.BusinessLogic/Services/UserListService.cs]: done

**Review by TREN-D — APPROVED** [review]

**Review by SergiyKozachok — APPROVED** [review]

**Review by eugent482 — APPROVED** [review]

**Pull request merged**